### ISSUE TITLE
Should the next release be 0.1.0, rather than 0.0.5 (API changes!)

### ISSUE THREAD

**Issue by christophfink** [r5py/r5py#188]
If we, as planned, refactor the ‘TravelTimeMatrixComputer’ code base into several classes, included one to calculate the expanded TTMs (as opposed to `breakdown=True` behaviour), our API changes break backwards-compatibility. 

According to semantic versioning rules, we should increase the *major* version. I argue for increasing the minor version, instead (but also not just the patch version we currently defined as our next milestone). Since we’re in such an early stage, bumping the major version seems over the top (and AFAIK `0.x.y` version tags signify unstable APIs and heavy development).

What are your opinions? Can we make the next release a `0.1.0`, and aim to have the majority of r5r’s API implemented in an equivalent, Pythonic, way?

**Comment by christophfink** [comment]
/polls rename to v0.1.0, stay with v0.0.5

**Comment by christophfink** [comment]
/polls 'rename to v0.1.0' 'stay with v0.0.5'

**Comment by christophfink** [comment]
hm, sorry for the many notification you all got now. I tried to use a poll, but the app does not seem to work: https://github.com/evenchange4/gh-polls-bot/issues/26 

If you have an opinion on whether the next release should be 0.1.0 or 0.0.5, could you comment in this thread until the end of the week?

**Comment by HTenkanen** [comment]
@christophfink: I also agree that we could perhaps go with the `v0.0.5` as was originally planned.
